docs/topics/db/models.txt

```diff
@@ -1356,6 +1356,20 @@ The general rules are:
    This sets things up so that the proxy model is an exact copy of the
    storage structure of the original model when data is saved.
 
+
+Proxy model and Unmanaged models ignore DB comment options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If Model has ``Meta.proxy=True``. or ``Meta.managed=False``,
+``Meta.db_table_comment=table comment...`` and ``Field.db_comment=comment...``
+has no effect.
+
+therefore it is recommended not to use ``Meta.proxy=True`` with
+``Meta.db_table_comment=table comment...`` and ``Field.db_comment=comment...``
+likewise it is recommended not to use ``Meta.managed=False`` with
+``Meta.db_table_comment=table comment...`` and ``Field.db_comment=comment...``.
+
+
```

Comment:
I don't think we need this, as it applies equally to any database changes in proxy models.

Reply:
While I agree that this `Proxy model and Unmanaged models ignore DB comment options` section probably shouldn't sit in this location in the docs, I do think it's useful to have this behavior documented in the section of the docs where Meta.db_table_comment and Field.db_comment are introduced. Because although experts will be familiar with the behavior of proxymodels, the interaction is not obvious for newer Django users.

Reply:
I agree that the location of this is not appropriate, and I will delete it because it seems to need rewrite this comment because  it is awkward sentence